### PR TITLE
EU4Converter tweaking

### PR DESCRIPTION
**Pull request by IhateTrains**
- removing some duplicates
- mapping WtWSMS governments for EU4Converter
- renaming Soissons to Romano-Gallic for EU4Converter
- adding Roman culture to Latin group for EU4Converter
- minor fixes

### CODE REVIEW

File: ChangeLog.txt [hunk 1]
```diff
@@ -567,4 +567,4 @@ MINOR:
 
 2014-12-31 v0.9
 ---------------
-- Initial BETA release of WtWSMS
+- Initial BETA release of WtWSMS
```
Comment (loup99) [inline ChangeLog.txt]: What happened here?

File: WTWSMS/eu4_converter/culture_table.csv [hunk 2]
```diff
@@ -262,14 +262,14 @@ bosporan;pontic;0;Pontic
...
-massalian;soissons;1;Greco-Gauls
+massalian;romano_gallic;1;Greco-Gauls
 helleno_thracian;thracian;1;Thracian
 #Mainland Celtic
 gallic;gaul;1;Gauls
 astures;gaul;1;Gauls
 galician;galician;0;Galician
 east_gallic;gaul;1;Gauls
-galatian;soissons;1;Romano-Gaulish
+galatian;romano_gallic;1;Romano-Gaulish
```
Comment (loup99) [inline WTWSMS/eu4_converter/culture_table.csv]: Another mapping which needs to change, Galatians are not Gallic.

File: ChangeLog.txt (hunk 1 above)
Comment (IhateTrains) [inline ChangeLog.txt]: IDK, there was a conflict, I resolved it and this appeared

File: WTWSMS/eu4_converter/culture_table.csv (hunk 2 above)
Comment (IhateTrains) [inline WTWSMS/eu4_converter/culture_table.csv]: Sure, in the next PR

File: ChangeLog.txt (hunk 1 above)
Comment (IhateTrains) [inline ChangeLog.txt]: I know, I removed an empty line

**Pull request merged**